tests/integration/components/public/speaker-item-test.js

```diff
@@ -0,0 +1,15 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('speaker-item', 'Integration | Component | speaker item');
```

Comment:
Should be 
`moduleForComponent('public/speaker-item', 'Integration | Component | public/speaker item');`